ros_buildfarm/bloom_status.py

```diff
@@ -76,41 +79,57 @@ def _get_package_info(g, distro_info):
                         headers = {}
                     tracks = _download_yaml(tracks_url, headers).get('tracks', {})
                     cached_tracks[url] = tracks
-                release_tag = tracks.get(distro, {}).get('release_tag', None)
-                if release_tag == ':{ask}' and 'last_release' in tracks[distro]:
-                    pkg[distro]['last_release'] = tracks[distro]['last_release']
+                dtracks = tracks.get(distro, {})
+                m = GITHUB_TOKEN_PATTERN.match(dtracks.get('vcs_uri', ''))
+                if m:
+                    distro_d['org'] = m.group(2)
+                upstream = dtracks.get('devel_branch', None)
```

Comment:
`None` is the default value for `get`'s default arg. But being explicit isn't bad either :)